content/api/servo-board/servos.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Servos
+---
+
+From the servo assembly, you can control the attached servos. By default, servos will be unpowered when your robot starts, and can freely rotate. Once you set a value, they then fix to that rotation. This also happens when you turn off your robot, or unplug your USB.
```

Comment:
I'd recommend saying "USB stick" instead of just "USB" (in case it gets misinterpreted as the USB connection to the servo assembly).